mex/common/public_api/transform.py

```diff
@@ -70,27 +75,22 @@ def transform_public_api_item_to_mex_model(
     dct_to_parse: dict[str, list[PublicApiFieldValueTypes]] = defaultdict(list)
     for value in api_item.values:
         field_name = value.fieldName
+        annotation = cls.model_fields[field_name].annotation
+        is_link = any(type_ is Link for type_ in [annotation, *get_args(annotation)])
+        is_text = any(type_ is Text for type_ in [annotation, *get_args(annotation)])
```

Comment:
```suggestion
        is_text = Text in (annotation, *get_args(annotation)])
```

why not that here? and: maybe this warrents a small util function?

Reply:
added util function in b7d22355a58e538c8144c7a8a9c8ab1907c1e4e9